Description: mount-archive.nemo_action: Use gnome-disk-image-mounter
 instead of gvfsd-archive.

Fixes #2811
Author: Michael Webster <miketwebster@gmail.com>
Last-Update: 2021-08-17
Origin: upstream, https://github.com/linuxmint/nemo/commit/b7bd80d71f076c3bc1a1060d39d7cd51c383a0af
---
 data/nemo-actions/mount-archive.nemo_action.in         | 4 ++--
 files/usr/share/nemo/actions/mount-archive.nemo_action | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/data/nemo-actions/mount-archive.nemo_action.in b/data/nemo-actions/mount-archive.nemo_action.in
index 9a084a447d..5fea593916 100644
--- a/data/nemo-actions/mount-archive.nemo_action.in
+++ b/data/nemo-actions/mount-archive.nemo_action.in
@@ -4,7 +4,7 @@ Name=Mount archive
 
 Comment=Mount %f to browse its contents
 
-Exec=/usr/lib/gvfs/gvfsd-archive file="%U"
+Exec=gnome-disk-image-mounter %U
 
 Selection=s
 
@@ -12,6 +12,6 @@ Mimetypes=application/x-7z-compressed;application/x-cd-image;application/x-cpio;
 
 Icon-Name=mount-archive-symbolic
 
-Dependencies=/usr/lib/gvfs/gvfsd-archive;
+Dependencies=gnome-disk-image-mounter;
 
 Selection=s
diff --git a/files/usr/share/nemo/actions/mount-archive.nemo_action b/files/usr/share/nemo/actions/mount-archive.nemo_action
index b2cdabe2b7..c9ca45b062 100644
--- a/files/usr/share/nemo/actions/mount-archive.nemo_action
+++ b/files/usr/share/nemo/actions/mount-archive.nemo_action
@@ -4,7 +4,7 @@ Name=Mount archive
 
 Comment=Mount %f to browse its contents
 
-Exec=/usr/lib/gvfs/gvfsd-archive file="%U"
+Exec=gnome-disk-image-mounter %U
 
 Selection=s
 
@@ -12,7 +12,7 @@ Mimetypes=application/x-7z-compressed;application/x-cd-image;application/x-cpio;
 
 Icon-Name=mount-archive-symbolic
 
-Dependencies=/usr/lib/gvfs/gvfsd-archive;
+Dependencies=gnome-disk-image-mounter;
 
 Selection=s
 Name[lt]=Prijungti archyvÄ…
